code/modules/mob/living/living.dm

```diff
@@ -703,6 +703,10 @@
 		var/obj/mecha/M = loc
 		loc_temp =  M.return_temperature()
 
+	else if(istype(loc, /obj/spacepod))
```

Comment:
I just gave up on doing that cause I couldn't think of how to implement it. thanks!